### DISCUSSION TITLE
Homepage colour brightness has changed/dulled

### DISCUSSION THREAD

**Discussion by cching95** [squidfunk/mkdocs-material#5548] [Q&A]
Has there been an update to mkdocs material theme which has changed the brightness of colours? or has something that was previously defaulted been switched off? In a older version of mkdocs material my homepage colours are much brighter but with the  current version I am using they have dulled slightly.

    - mkdocs-material==9.1.8
    - mkdocs-material-extensions==1.1.1

<img width="200" alt="Screenshot 2023-05-22 at 14 07 32" src="https://github.com/squidfunk/mkdocs-material/assets/73163191/8e8c00e1-93a5-43b1-b08b-90ea17bbeb9c">
<img width="231" alt="Screenshot 2023-05-22 at 14 07 12" src="https://github.com/squidfunk/mkdocs-material/assets/73163191/e89747e0-f15a-46ca-8fb7-ecc61ad84462">

**Comment by squidfunk** [comment]
I'm pretty sure we didn't change anything here, but gradients are only used for the custom homepage anyways, not in Material for MkDocs itself. Thus, even if a change might have happened at some point (with a library update or so), it's not a bug.
